src/passwordReset.js

```diff
@@ -0,0 +1,30 @@
+const nodemailer = require("nodemailer");
+
+exports.post = (req, res) => {
+  const transporter = nodemailer.createTransport({
+    service: "gmail",
+    auth: {
+      user: "sleuth.reset@gmail.com",
+      pass: "sleuth123"
+    }
+  });
```

Comment:
use environmental variables this email and password is tracked in github, please change the password